Documentation/security/network/proxy/envoy.rst

```diff
@@ -454,36 +459,36 @@ as an example:
                     "file": reqData.file,
                 },
             },
-      })  
+      })
 
 Step 12: Manual Testing
 =======================
 
-Find the standard docker container for running the protocol server.  Often the same image also has a CLI client that you can use as a client. 
+Find the standard docker container for running the protocol server.  Often the same image also has a CLI client that you can use as a client.
 
 Start both a server and client container running in the cilium dev VM, and attach them to the already created “cilium-net”.  For example, with Cassandra, we run:
 
 .. code-block:: shell-session
 
     docker run --name cass-server -l id=cass-server -d --net cilium-net cassandra
 
-    docker run --name cass-client -l id=cass-client -d --net cilium-net cassandra sh -c 'sleep 3000' 
- 
+    docker run --name cass-client -l id=cass-client -d --net cilium-net cassandra sh -c 'sleep 3000'
 
-Note that we run both containers with labels that will make it easy to refer to these containers in a cilium 
-network policy.   Note that we have the client container run the sleep command, as we will use 'docker exec' to 
-access the client CLI.  
 
-Use ''cilium-dbg endpoint list'' to identify the IP address of the protocol server.  
+Note that we run both containers with labels that will make it easy to refer to these containers in a cilium
+network policy.   Note that we have the client container run the sleep command, as we will use 'docker exec' to
+access the client CLI.
+
+Use ''cilium-dbg endpoint list'' to identify the IP address of the protocol server.
```

Comment:
If we changed that line we should also fix the quotes (should be back quotes instead of straight quotes).

Reply:
fixed